flight/targets/naze32/board-info/board_hw_defs.c

```diff
@@ -0,0 +1,904 @@
+/**
+ ******************************************************************************
+ * @addtogroup TauLabsTargets Tau Labs Targets
+ * @{
+ * @addtogroup CopterControl OpenPilot naze support files
+ * @{
+ *
+ * @file       board_hw_defs.c 
+ * @author     The OpenPilot Team, http://www.openpilot.org Copyright (C) 2011.
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2013
+ * @brief      Defines board specific static initializers for hardware for the 
+ *             CopterControl board.
+ * @see        The GNU Public License (GPL) Version 3
+ * 
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+ 
+#include <pios_config.h>
+#include <pios_board_info.h>
+
+#if defined(PIOS_INCLUDE_LED)
+
+#include <pios_led_priv.h>
+static const struct pios_led pios_leds[] = {
+	[PIOS_LED_HEARTBEAT] = {
+		.pin = {
+			.gpio = GPIOB,
+			.init = {
+				.GPIO_Pin   = GPIO_Pin_3,
+				.GPIO_Mode  = GPIO_Mode_Out_PP,
+				.GPIO_Speed = GPIO_Speed_50MHz,
+			},
+		},
+		.remap = GPIO_Remap_SWJ_JTAGDisable,
+	},
+	[PIOS_LED_ALARM] = {
+		.pin = {
+			.gpio = GPIOB,
+			.init = {
+				.GPIO_Pin   = GPIO_Pin_4,
+				.GPIO_Mode  = GPIO_Mode_Out_PP,
+				.GPIO_Speed = GPIO_Speed_50MHz,
+			},
+		},
+	},
+};
+
+static const struct pios_led_cfg pios_led_cfg = {
+	.leds     = pios_leds,
+	.num_leds = NELEMENTS(pios_leds),
+};
+
+const struct pios_led_cfg * PIOS_BOARD_HW_DEFS_GetLedCfg (uint32_t board_revision)
+{
+	return &pios_led_cfg;
+}
+
+#endif	/* PIOS_INCLUDE_LED */
+
+#if defined(PIOS_INCLUDE_SPI)
+
+#include <pios_spi_priv.h>
+
+/* Flash/Accel Interface
+ * 
+ * NOTE: Leave this declared as const data so that it ends up in the 
+ * .rodata section (ie. Flash) rather than in the .bss section (RAM).
+ */
+void PIOS_SPI_generic_irq_handler(void);
+void DMA1_Channel4_IRQHandler() __attribute__ ((alias ("PIOS_SPI_generic_irq_handler")));
+void DMA1_Channel5_IRQHandler() __attribute__ ((alias ("PIOS_SPI_generic_irq_handler")));
+static const struct pios_spi_cfg pios_spi_generic_cfg = {
```

Comment:
If it's useful work towards supporting the flash chip, which it looks like it is, and doesn't break anything... I'd say we keep it.  Having a low-end platform that can log uavos would be awesome.